stripe/src/main/java/com/stripe/android/view/CardInputWidget.java

```diff
@@ -31,13 +31,25 @@
 import com.stripe.android.util.StripeTextUtils;
 
 import java.util.HashMap;
+import java.util.Locale;
 import java.util.Map;
 
 /**
  * A card input widget that handles all animation on its own.
  */
 public class CardInputWidget extends LinearLayout {
 
+    public static final Map<String , Integer> BRAND_RESOURCE_MAP =
```

Comment:
Customer wanted to be able to hit the icons. Harmless, easy to change, so why not?